libr/main/radare2.c

```diff
@@ -1489,5 +1489,6 @@ R_API int r_main_radare2(int argc, const char **argv) {
 	free (file);
 	r_cons_free ();
 	LISTS_FREE ();
+	R_FREE (pfile);
```

Comment:
I'm not 100% sure this is correct. In some paths in this function there is `pfile = file` and file is already freed. So I'm not sure this won't cause double frees in some cases.

Reply:
check now, the free(file) was a bit weird. im not 100% sure either

Reply:
still not super sure... it could be that now `file` will leak :D but the pR is ok to me. let's see. this code is a bit messy and hard to follow.